test/workflows/components/workflows.libsonnet

```diff
@@ -207,16 +207,16 @@
                     template: "setup-cluster",
                   },
                 ],
-                [
-                  {
-                    name: "build-kubebench-controller",
-                    template: "build-kubebench-controller",
-                  },
-                  {
-                    name: "build-kubebench-examples",
-                    template: "build-kubebench-examples",
-                  },
-                ],
+                // [
```

Comment:
we still need to build the images. this should not affect the tests to pass.

Reply:
@xyhuang Done it.